src/feature/playerhistory/playerhistory.controller.ts

```diff
@@ -0,0 +1,86 @@
+import {
```

Comment:
Rename your feature from ‘playerhistory’ to ‘history’. Either ‘player-history’ or ‘history’ will do. But shorter names will make your life a bit easier. Remember to rename the class names too.